ferveo-python/src/lib.rs

```diff
@@ -146,7 +146,6 @@ impl Keypair {
         to_py_bytes(self.0)
     }
 
-    #[getter]
     pub fn public_key(&self) -> PublicKey {
```

Comment:
Why does it need to be callable?

Reply:
I actually understand the getter is conventional in this codebase... the main reason for this change is due to convention in `nucypher/nucypher` and `nucypher/rust-umbral` where we employ a generic key-handling pattern that expects that a keypair's public key field is a callable.  

If it's a python property instead then we'll end up building a shim/wrapper around `ferveo_py.Keypair` in order to avoid modifying the public key attrs defined in `rust-umbral` both of which I'd like to avoid 😅

Reply:
Ok, I see. Thanks for sharing. What is the convention you're using for callable, is it just `public_key` or something like `get_public_key`?

Reply:
Yep, it's just `Keypair.public_key()`.  The nucypher powers and keypairs internal APIs can then handle both Umbral and Ferveo generated keys in the same generic patterns/objects.